### ISSUE TITLE
enhanced/impaired conditioned taste aversion behavior

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3352]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. enhanced conditioned taste aversion behavior
2. impaired conditioned taste aversion behavior

>2. Synonyms (e.g., absent spleen)

1. increased conditioned taste aversion behavior
2. increased conditioned taste aversion behavior

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. increase in the ability of an animal to learn to avoid food sources that were previously associated with aversive internal stimuli or sickness such as a drug that induces sickness
2. decrease in the ability of an animal to learn to avoid food sources that were previously associated with aversive internal stimuli or sickness such as a drug that induces sickness

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID:11958542

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

abnormal conditioned taste aversion behavior (MP:0002805)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5356-4174
--

**Comment by sbello** [comment]
MP:0020968 - enhanced
MP:0020969 - impaired
not yet public
